lib/options.js

```diff
@@ -170,7 +171,7 @@ module.exports = {
         description: 'Select whether to include authentication parameters in the example request',
         external: true,
         usage: ['CONVERSION'],
-        supportedIn: [VERSION30, VERSION31]
+        supportedIn: [VERSION20, VERSION30, VERSION31]
```

Comment:
There are other options present which should be supported for `2.0`, i.e. `disableOptionalParameters` and `keepImplicitHeaders` were missed. Can we make sure all conversion-related options supported by 3.0 is supported here?